spikeinterface/core/waveform_extractor.py

```diff
@@ -197,7 +199,7 @@ def get_waveforms(self, unit_id, with_index=False):
         Returns
         -------
         wfs: np.array
-            The returned waveform (num_spikes, num_channels, num_samples)
+            The returned waveform (num_spikes, num_samples, num_channels)
```

Comment:
This is an error in orientation. Even in tests its this changed dimension order.